<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Угадывание чисел</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>235</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество элементов из БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphResultPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>251</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести графический результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>232</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите 6-ти значное число</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kmeansPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Разбиение на 4 кластера</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить анализ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Дополнительное число</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Число из списка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Результат анализа</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="countSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dopNumberSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100000</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>123456</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
